.doc_gen/metadata/lambda_metadata.yaml

```diff
@@ -76,3 +76,17 @@ lambda_Scenario_DeployInvokeFunction:
                 - python.example_code.lambda.Scenario_DeployInvokeFunction_Demo
   services:
     lambda: {CreateFunction, DeleteFunction, Invoke}
+cross_LambdaForBrowser:
+  title: Invoke a &LAM; function from the browser
+  title_abbrev: Invoke a &LAM; function from browser
+  synopsis: invoke an &LAMlong; function from a browser.
```

Comment:
Standardize on "from a browser" for these three.

Reply:
done